automatic/sysinternals/tools/chocolateyInstall.ps1

```diff
@@ -21,6 +21,23 @@ if (Is-NanoServer) {
   $packageArgs.checksum = '7b6fab7f0ada0bc2b0748f3bc29e93846869f9dd175a0ca070bbebc07ee5de09'
  }
 
+if (-Not $pp.NoStartMenu) {
+  Write-Verbose "Creating start menu entry"
+  set-alias 7z $Env:chocolateyInstall\tools\7z.exe
+  $linkfiles = get-childitem "$installDir" -include "*.exe" -recurse
+  $startmenufolder = "$($env:ProgramData)\Microsoft\Windows\Start Menu\Programs\Sysinternals Suite"
+  Add-Content "$toolsPath\chocoUninstall.ps1" -Value "Remove-Item `"$($startmenufolder)`" -Recurse -Force -ea ignore"
```

Comment:
Do not add text directly to the uninstall script, rather add the paths to a separate text file that the uninstall script can read.